docs/ThemeAggregation.md

```diff
@@ -0,0 +1,64 @@
+# Theme Aggregation
+
+Terra Toolkit provides a built-in mechanism for aggregating themes.
+
+By default, the terra-toolkit webpack configuration enables theme aggregation when it detects the presence of a `theme.config.js` file.
```

Comment:
It still feels like there should be a flag you set in your webpack configuration that determines if themes should be aggregated or not. What if for some reason they already have a theme.config.js which does something else?

If you do need to do the approach of having a file present, could we namespace the file (terra_theme.config.js)?

Reply:
Is there a scenario you can think of where a user would have an intentional theme aggregation config file and not want to aggregate themes?

We could definitely look into name-spacing the file. It just feels a little redundant to have a configuration file and then also be required to opt into using it.

Reply:
We wouldn't be the first one to do that. I know there are several PostCSS plugins that either allow you to define their configuration inside the webpack file or inside a separate file itself.

Nevertherless, I think having a namespaced config file is appropriate.

Reply:
Do those configurations require you to do both? Declare the file and also provide a webpack configuration option to use that file? 

I'm not sure we should enforce a `terra` specific namespace on project files, but we could do something like `theme-aggregation.config.js` or something similar. It's a little wordy but I wouldn't expect their to be any collisions.

Reply:
I think the PostCssAssetsPlugin either wants the config inside the webpack file, or a configuration file in your repo, and then in the webpack config, you provide a path to the configuration file.

Reply:
theme-aggregation is fine as well.

Reply:
The configuration file has been updated to be `terra-theme.config.js`